SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -415,7 +416,21 @@ - (OWSSignalServiceProtosDataMessageBuilder *)dataMessageBuilder
     
     OWSSignalServiceProtosDataMessageBuilder *builder = [OWSSignalServiceProtosDataMessageBuilder new];
     [builder setTimestamp:self.timestamp];
-    [builder setBody:self.body];
+
+
+    if ([self.body lengthOfBytesUsingEncoding:NSUTF8StringEncoding] <= kOversizeTextMessageSizeThreshold) {
```

Comment:
Be more defensive about enforcing this limit in production in case we missed another entry point.